crypto/objects/o_names.c

```diff
@@ -157,11 +174,13 @@ const char *OBJ_NAME_get(const char *name, int type)
 {
     OBJ_NAME on, *ret;
     int num = 0, alias;
+    const char *value = NULL;
 
     if (name == NULL)
         return (NULL);
-    if ((names_lh == NULL) && !OBJ_NAME_init())
+    if (!OBJ_NAME_init())
         return (NULL);
+    CRYPTO_THREAD_read_lock(lock);
```

Comment:
Probably not. It was used by the since removed `-stats` flag for the `errstr` command. We removed it, and I've never seen anything notice.